demo/docbot/cluster_bootstrap.py

```diff
@@ -119,40 +120,41 @@ def initialize_model_group(client: MLClient) -> None:
 
 
 
-def initialize_connector(client: MLClient) -> None:
+def initialize_connector(client: MLClient, connector_name="Cohere Connector", connector_data=None) -> None:
     """
     Args:
         Client: OpenSearch
     Returns:
         returns None if connector was created or is exists else raises Exception
     """
     # Once again, only allowing for the Cohere Connector. We can change this later on.
-    connector_data = {
-        "name": "Cohere Connector",
-        "description": "External connector for connections into Cohere",
-        "version": "1.0",
-        "protocol": "http",
-        "credential": {
-            "cohere_key": COHERE_KEY
-        },
-        "parameters": {
-            "model": "embed-english-v2.0",
-            "truncate": "END"
-        },
-        "actions": [{
-            "action_type": "predict",
-            "method": "POST",
-            "url": "https://api.cohere.ai/v1/embed",
-            "headers": {
-                "Authorization": "Bearer ${credential.cohere_key}"
+    if connector_data is None:
+        connector_data = {
+            "name": "Cohere Connector",
```

Comment:
We should probably be using the variable name here. Then in the description below we can use f-strings again to get the connector name to show correctly there.